Hunting Queries/SecurityEvent/hunt_LOLBins.yaml

```diff
@@ -0,0 +1,31 @@
+id: 4a6030de-c11c-4206-b4ea-cd005ebf80a2
+name: LOLBins and LOLScripts
+description: |
+  'This query identifies Microsoft-signed Binaries and Scripts that are not system initiated. This technique is commonly used in phishing attacks'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+tactics:
+  - Execution
+relevantTechniques:
+  - T1218
+  
+query: |
+  //This query uses sysmon data depending on table name used this may need updataing
+  let Bin = externaldata(Binary: string,values: dynamic) [@"https://github.com/sonnyakhere/LOLBAS_to_CSV/blob/main/lolbas.csv"]
```

Comment:
Hey, instead of reading HTML version and parsing out executables manually , you can render raw github version with externaldata... something like this

`let Bin = externaldata(Binary: string) [@"https://raw.githubusercontent.com/sonnyakhere/LOLBAS_to_CSV/main/lolbas.csv"]
  with (format="csv", ignoreFirstRecord=True);
  Bin`

Reply:
Thanks! Done that.